assets/src/edit-story/app/media/media3p/providerReducer.js

```diff
@@ -21,8 +21,13 @@ import commonReducer, {
   INITIAL_STATE as COMMON_INITIAL_STATE,
 } from '../common/reducer';
```

Comment:
The state attributes should be defined on their own for categories, as there are attributes specific to media, such as 'media'.

Reply:
Ignore me, this has nothing to do with your changes :)